app.js

```diff
@@ -258,7 +265,7 @@ app.get("/login", function (req, res, next) {
     if (_.isString(req.session.relayState)) {
       req.query.RelayState = req.session.relayState;
     }
-    spOptions.forceAuthn = (req.query.forceauthn !== undefined);
+    //spOptions.forceAuthn = (req.query.forceauthn !== undefined); go based off the parameter on startup?
```

Comment:
oh the forceauthn stuff isn't that important for the critical flows right now. This was just from me trying to have it being set in /settings